docs/sources/reference/components/mimir/mimir.alerts.kubernetes.md

```diff
@@ -147,6 +148,33 @@ The `operator` argument should be one of the following strings:
 
 The `values` argument must not be provided when `operator` is set to `"Exists"` or `"DoesNotExist"`.
 
+### `alertmanagerconfig_matcher`
+
+The `alertmanagerconfig_matcher` block describes the strategy used by AlertmanagerConfig objects to match alerts in the routes and inhibition rules.
+
+Depending on how this block is configured, the final Alertmanger config will have different [matchers][] in its [route][] section.
+
+[matchers]: https://prometheus.io/docs/alerting/latest/configuration/#matcher
+[route]: https://prometheus.io/docs/alerting/latest/configuration/#route
+
+The following arguments are supported:
+
+| Name                     | Type     | Description                                                                                                          | Default         | Required                                                              |
+| ------------------------ | -------- | -------------------------------------------------------------------------------------------------------------------- | --------------- | --------------------------------------------------------------------- |
+| `strategy`               | `string` | Strategy for adding matchers to AlertmanagerConfig CRDs.                                                             | `"OnNamespace"` | no                                                                    |
+| `alertmanager_namespace` | `string` | Namespace to use when `alertmanagerconfig_matcher_strategy` is set to `"OnNamespaceExceptForAlertmanagerNamespace"`. |                 | only when `strategy` is `"OnNamespaceExceptForAlertmanagerNamespace"` |
```

Comment:
@clayton-cornell is it ok if the "Required" column is populated like this? Is it too verbose? I couldn't run `make docs` locally due to a docker error so I can't visualise it.